codebase/analysis/src.idl/lib/igrf.1.0/igrflib_v2.pro

```diff
@@ -1128,7 +1128,7 @@ end
 function geod2geoc, lat, lon, alt
   common IGRF_v2_Com
 
-  a = 6378.1370d;             /* semi-major axis */
+  a = !CONST.R_earth/1000;    /* semi-major axis */
```

Comment:
The WGS-84 ellipsoid reference has an exact definition for the semi-major axis which should probably not be modified here or elsewhere (https://en.wikipedia.org/wiki/World_Geodetic_System#Defining_Parameters).

Reply:
Oops, didn't even cross my mind. I agree.